src/appleseed.shaders/include/appleseed/math/as_math_helpers.h

```diff
@@ -34,7 +34,6 @@
 float sqr(float x) { return x * x; }
 vector sqr(vector x) { return x * x; }
 
-
```

Comment:
Block comments should be preceded by two blank lines so I would suggest to revert these changes.